app/App.tsx

```diff
@@ -1,11 +1,44 @@
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
+import React = require('react');
```

Comment:
Can you revert this file back to what it was on the `main` branch? You can move the test event data into `EventList.tsx` if you'd like to keep it for testing later.